duckduckgo/iOS
# Add inspectable webview support

Task/Issue URL: https://app.asana.com/0/414709148257752/1204241755214739/f
Tech Design URL:
CC:

**Description**:

This PR adds support for the new `inspectable` flag on `WKWebView`. This allows internal users to turn on inspectability and gain access to the remote web inspector via Safari.

Note that this feature does not use the internal feature flag, it is tied exclusively to the Debug menu (which is itself internal only as far as end users are concerned) as the Debug menu can be accesssed when connected to Xcode even when you aren't technically in Internal mode.



**Steps to test this PR**:
1. Run a clean install of the app, check that the webview is not inspectable by connecting a physical device to macOS and checking that the webview cannot be inspected via Safari's `Develop` menu
2. Go into the debug menu of the app, turn on the `Enable Inspectable Webviews` option, and then verify that you can now inspect the current open tab using Safari's `Develop` menu
3. Turn the setting off again and validate that inspectability support is disabled



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
